app/actors/sufia/create_with_files_actor.rb

```diff
@@ -46,10 +46,32 @@ def uploaded_files
       def mark_inactive
         return true unless Flipflop.enable_mediated_deposit?
         curation_concern.state = inactive_uri
+        send_inactive_message
```

Comment:
Sending notifications in this class violates the Single Responsibility Principal, and it should be in the background because it's slow. Consider making a subclass of `ContentEventJob` for this (see https://github.com/projecthydra/sufia/blob/master/app/jobs/content_update_event_job.rb#L2 as an example)